gator/report.py

```diff
@@ -19,7 +19,7 @@
 TEXT = "output_text"
 JSON = "output_json"
 
-ARROW = "↦"
+ARROW = "🠆"
```

Comment:
Encoding may have messed up somewhere -- unicode arrow turned into an undefined unicode codepoint

Reply:
Thanks @Michionlion, I will look into this. Perhaps I have to specify the unicode type at the top of the file? This works correctly for me with my shell, in tmux, etc. Do you have a specific suggestion on how to resolve the error? If not, then I will investigate further after I finish another feature.

Reply:
with the default vim configuration the arrow is saved just fine for me... It was also saved when you first committed. There may be a tool that you used, perhaps for formatting, that either needs to support unicode, or needs an option flipped for that.

Reply:
Also, @Michionlion, just as a note, I will not merge this pull request until I have resolve this issue.

Reply:
Thanks for investigating this @Michionlion. I am now using `U+2794`. Let me know if this does not work correctly on your system. I was aiming to use a "heavy" arrow so that it could easily be seen on a wide variety of terminals. If this one seems to not work, there are, of course, several others to try as well.